crates/telio-relay/src/derp/proto.rs

```diff
@@ -244,6 +244,7 @@ pub async fn start_write<W: AsyncWrite + Unpin>(
                     break;
                 }
             }
+            else => break,
```

Comment:
else not possible, there are no additional hands for else, as it receives option's and then matches on them

Reply:
Maybe we should leave it, for the sake of to have "else-less" selects ..?

Reply:
@matislovas it sounds like adding `_ => {}` to every `match`, imho weird idea. When it is exhaustive without that it sounds like a better design.

Reply:
@stalowyjez  but in that case, will you be able to lint out such specific cases, where it is prone to panic? It is similar to `match`, except that `match` does not panic :sweat_smile: